### ISSUE TITLE
Project manager interface slides offscreen on Android editor due to labels being too long

### ISSUE THREAD

**Issue by tremisabdoul** [godotengine/godot#60728]
### Godot version

3.5 alpha 4

### System information

Android 6.0.1 - Samsung galaxy S5 neo - (Resolution: 1920x1080px)

### Issue description

Hello!
I've seen an bug on the new Android Godot Editor.
I've install and run the 3.5 alpha 4 version but i can't use it. 
The "Menu" of the editor didn't go fully on the right of the window (the scan, import, open, etc. are inaccessible). I think it's an screen calibration problem. 

// Sorry for the language errors i'm french

Btw ty and gg guys for your work on godot :)
Have a nice day!

### Steps to reproduce

Install 3.5 alpha 4 APK version on mobile and run it

### Minimal reproduction project

No code needed

*Bugsquad edit (keywords for easier searching): cut off, cutoff, trim, trimmed*

**Comment by tremisabdoul** [comment]
![Screenshot_20220503-163259](https://user-images.githubusercontent.com/75373856/166481617-8e85d4dd-a450-4a9c-a4d6-f19b8bedcb8a.png)
Little screenshot to illustrate it.
The top bar is calibrate but not the body and the border.

**Comment by tremisabdoul** [comment]
![Screenshot_20220503-172224](https://user-images.githubusercontent.com/75373856/166484012-010e6028-30e6-4277-8473-9d1948fdb720.png)
![Screenshot_20220503-163259](https://user-images.githubusercontent.com/75373856/166481617-8e85d4dd-a450-4a9c-a4d6-f19b8bedcb8a.png)
Little screenshots to illustrate it.
The top bar is calibrate but the body isn't.

**Comment by Calinou** [comment]
This is due to the same cause as https://github.com/godotengine/godot/issues/31133.

You can alleviate this by changing the editor language to English, but you can currently only achieve this by changing the language in the Android system settings due to https://github.com/godotengine/godot/issues/60353.

**Comment by tremisabdoul** [comment]
Ok thank you (sorry i've didnt saw theres issues).

**Comment by Calinou** [comment]
For reference, here's what the project manager window looks like when you set its minimum size to `0, 0` (by recompiling the editor for this purpose):

https://user-images.githubusercontent.com/180032/166718873-2921d7a3-b224-4a78-b50d-98b4829d5d5d.mp4

**Comment by akien-mga** [comment]
> Check if project list doesn't impose a minimum width on its own – if it is, it probably shouldn't.

It definitely looks like it does, as it stops shrinking after a while even though the text clips and the other elements shouldn't impose a wider min width.

**Comment by JoluMorsanDev** [comment]
> For reference, here's what the project manager window looks like when you set its minimum size to `0, 0` (by recompiling the editor for this purpose):
> 
>  simplescreenrecorder-2022-05-04_17.41.10.mp4 
> Things that can be done to reduce the minimum width required:
> 
> * Make the search LineEdit shorter on small window sizes.
> * Check if project list doesn't impose a minimum width on its own – if it is, it probably shouldn't.
> * Display project icons at a lower size (or even hide them entirely) on very small window sizes.
> * Hide the version Label on very small window sizes (or truncate it).
> 
> Also, the About button should be hidden at small window sizes, as its dialog will probably look broken in those situations. In fact, the About dialog may not be able to be closed if it fills the whole screen, no hardware keyboard is connected to press Escape and the Close button isn't visible.

I don't understand, is there a solution then? Or we have to wait? I have no problem but I want to know if I can do something (unless is very complex)

**Comment by Zireael07** [comment]
@JoluMorsanDev: Can you compile Godot for Android or does that qualify as "very complex" for you?

**Comment by JoluMorsanDev** [comment]
@Zireael07 I don't know how to do it, but I can try, where do I find instructions?

**Comment by Calinou** [comment]
> @Zireael07 I don't know how to do it, but I can try, where do I find instructions?

See [Compiling for Android](https://docs.godotengine.org/en/stable/development/compiling/compiling_for_android.html) in the documentation. You will need access to a PC to compile for Android. Also, remember to pass `tools=yes` on the SCons build options.

That said, if you don't make the required code changes explained above, compiling the editor for Android will *not* resolve this problem.

**Comment by akien-mga** [comment]
CC @m4gr3d. While we can and should improve the project manager so that it can support smaller sizes, this issue mostly shows that the choice of scaling ratio based on device DPI is not working well currently. On this device it shouldn't choose a scaling ratio that makes things so big.

The specs for that device seem to be:
```
Size | 5.1 inches, 71.7 cm2 (~69.6% screen-to-body ratio)
Resolution | 1080 x 1920 pixels, 16:9 ratio (~432 ppi density)
```
https://www.gsmarena.com/samsung_galaxy_s5_neo-6506.php

**Comment by JoluMorsanDev** [comment]
> > @Zireael07 I don't know how to do it, but I can try, where do I find instructions?
> 
> See [Compiling for Android](https://docs.godotengine.org/en/stable/development/compiling/compiling_for_android.html) in the documentation. You will need access to a PC to compile for Android. Also, remember to pass `tools=yes` on the SCons command line, then use `./gradlew generateGodotEditor` within `platform/android/java/` to generate an APK.
> 
> That said, if you don't make the required code changes explained above, compiling the editor for Android will _not_ resolve this problem.
> 
> If you don't know how to fix this, this will take a while to be resolved, as most contributors have other priorities right now (and don't have time to dedicate to the Android editor).

I work on pc so I don't have any problem wating, I just wanted to test it on android, anyways I'll try the compilation, thank you.

**Comment by akien-mga** [comment]
This should have been pretty much fixed by #61210 and #61595 in 3.5.

If some similar issues are still reproducible in 3.5.x or later, please open a new issue.
